Scripts/models/logit.py

```diff
@@ -607,15 +607,26 @@ def calc_prob(self, impedance):
                 Choice probabilities
         """
         mode_expsum = self._calc_mode_util(impedance)
-        logsum = {"logsum": mode_expsum}
-        dest_expsum, dest_exps = self._calc_dest_util("logsum", logsum)
+        dest_exps = self._calc_dest_util("logsum", {"logsum": mode_expsum})
+        try:
+            dest_expsum = dest_exps.sum(1)
+        except ValueError:
+            dest_expsum = dest_exps.sum()
+        logsum = pandas.Series(
+            numpy.log(dest_expsum), self.purpose.zone_numbers,
+            name=self.purpose.name)
+        self.accessibility = {"all": logsum}
+        self.zone_data._values[self.purpose.name] = logsum
         prob = {}
         dest_prob = dest_exps.T / dest_expsum
         for mode in self.mode_choice_param:
             mode_prob = (self.mode_exps[mode] / mode_expsum).T
```

Comment:
This produces NAs, I think `mode_prob = numpy.divide(self.mode_exps[mode], mode_expsum, out=numpy.zeros_like(self.mode_exps[mode]), where=mode_expsum!=0).T` fixes this.